doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1106,8 +1107,10 @@ Response:
       + Members
           + keyValues - when used, the request payload uses the `keyValues` simplified entity
             representation. See "Simplified Entity Representation" section for details.
+          + upsert - when used, entity is updated if already exits (otherwise this situation causes
+            a 422 Unprocessable Entity error).
 
-+ Response 201
++ Response 201 204
```

Comment:
Not sure how this should be included in the .apib

* Enumerating the possible 2xx codes? (as it is now)
* Using two `+ Response` blocks, one for 201 the other for 204?
* Using only 201 as an example (as it was before)?

@jmcanterafonseca some advice on this would be great, please.

Reply:
Better to add a new response code

Reply:
Does .apib accept to have something like this?

```
+ Response 201
   ...
+ Response 204
  ...
```

(Maybe is a naïve question, but we haven't used that pattern in the .apib up to now :)

Reply:
New block added in 6566b8aad31afa25fed44b9b8a5aec3661ef71f1... although I'm not sure if works this way (as asked in my previous comment).